flake.nix

```
@@ -30,6 +30,7 @@
       lenovo-ideapad-z510 = import ./lenovo/ideapad/z510;
       lenovo-thinkpad-e495 = import ./lenovo/thinkpad/e495;
       lenovo-thinkpad-l13 = import ./lenovo/thinkpad/l13;
+      lenovo-thinkpad-l14-amd = import ./lenovo/thinkpad/l14-amd;
```

As you said we should move this to:

```suggestion
      lenovo-thinkpad-l14-amd = import ./lenovo/thinkpad/l14/amd;
```

---

Done.